crypto/packet.c

```diff
@@ -484,6 +488,17 @@ int WPACKET_get_total_written(WPACKET *pkt, size_t *written)
     return 1;
 }
 
+int WPACKET_get_total_attempted_written(WPACKET *pkt, size_t *written)
```

Comment:
This is IMO unnecessary. Instead use the WPACKET_init_null() for the NULL destination buffer to get the number of bytes needed in the buffer.

Reply:
This would work for a NULL destination however the caller also needs to return the correct length if the writes overflow.  I'd need to write everything twice in that case.